tests/test-cases/Group1-Docker-Commands/1-03-Docker-Images.robot

```diff
@@ -61,7 +61,7 @@ No-trunc images
     Should Be True  ${count} >= 3
     @{lines}=  Split To Lines  ${output}
     @{line}=  Split String  @{lines}[2]
-    Length Should Be  @{line}[2]  64
+    Length Should Be  @{line}[2]  71
```

Comment:
In addition to this, can we explicitly verify that the prefix is what we expect (`sha256:`)? Or perhaps verify that the value matches a regular expression (e.g., `sha256:[0-9a-f]{64}`)? I'm not sure that the significance of 71 will be clear if this test begins failing in the future.

Reply:
done.